assets/src/components/detourListPage.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react"
+import { Table } from "react-bootstrap"
+import { RoutePill } from "./routePill"
+import { useCurrentTimeSeconds } from "../hooks/useCurrentTime"
+import { timeAgoLabel } from "../util/dateTime"
+import useScreenSize from "../hooks/useScreenSize"
+
+export const DetourListPage = () => {
+  const fakeData = [
+    {
+      number: 45,
+      direction: "Outbound",
+      name: "Franklin Park via Ruggles Station",
+      intersection: "John F. Kennedy St & Memorial Drive",
+      activeSince: 1722372950,
+    },
+    {
+      number: 83,
+      direction: "Inbound",
+      name: "Central Square",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1722361948,
+    },
+    {
+      number: 83,
+      direction: "Outbound",
+      name: "Rindge Ave",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1721361948,
+    },
+    {
+      number: 45,
+      direction: "Outbound",
+      name: "Franklin Park via Ruggles Station",
+      intersection: "John F. Kennedy St & Memorial Drive",
+      activeSince: 1722372950,
+    },
+    {
+      number: 83,
+      direction: "Inbound",
+      name: "Central Square",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1722361948,
+    },
+    {
+      number: 83,
+      direction: "Outbound",
+      name: "Rindge Ave",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1721361948,
+    },
+    {
+      number: 45,
+      direction: "Outbound",
+      name: "Franklin Park via Ruggles Station",
+      intersection: "John F. Kennedy St & Memorial Drive",
+      activeSince: 1722372950,
+    },
+    {
+      number: 83,
+      direction: "Inbound",
+      name: "Central Square",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1722361948,
+    },
+    {
+      number: 83,
+      direction: "Outbound",
+      name: "Rindge Ave",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1721361948,
+    },
+  ]
+
+  const epochNowInSeconds = useCurrentTimeSeconds()
+  const screenSize = useScreenSize()
+
+  return (
+    <>
+      <Table hover className="c-detour-table">
+        <thead>
+          <tr>
+            <th>Route</th>
+            {["desktop", "tablet"].includes(screenSize) && (
+              <>
+                <th>Starting Intersection</th>
+                <th>On detour since</th>
+              </>
+            )}
```

Comment:
I'm fairly certain we can do this entirely within CSS, is there a better reason to do it like this?

Reply:
Ah yes yes! I noticed this in your review of [Josh's PR](https://github.com/mbta/skate/pull/2716). 

The problem is that our [Skate-set breakpoints](https://github.com/mbta/skate/blob/a9a862e1a7dc4e51f1d4861978dbd8d22eab148f/assets/css/_skate_ui.scss#L1) are different from the [bootstrap breakpoints](https://getbootstrap.com/docs/5.0/layout/breakpoints/). This might need to be a deeper conversation with design to determine whether our chosen breakpoints are preferred to Bootstrap's, and whether we should override the Bootstrap defaults. For now, it seems Skate primarily uses our own custom breakpoints instead of the Bootstrap ones, so I'd lean towards consistency for now?

Reply:
you can still do this within CSS with our own breakpoints, which addresses the consistency issue.